README.md

```diff
@@ -15,8 +15,8 @@ It links up with Google Sheets to generate an at-a-glance list of all the events
 ### Getting Google Sheets credentials
 
 1. Talk to someone on the team (currently Benji) who can give you Google sheets credentials, and ask for them.
```

Comment:
This sentence isn't needed as you don't need anyone else to do anything to raise the hyp request 🙂

Reply:
I think this was originally in there as when you are locally testing the bot, you need a separate sheet with it's own credentials and access (which benji would need to make). Do we still want that or is it better to just have one testing sheet for everyone to use?

Reply:
Ah I see! Is there any reason why people can't share a testing sheet? Because that would make it a bit easier for a new person to ramp up. What do you think @benjimarshall? 

If we want to keep using separate sheets then we should document here how to create a new one for a new user. Basically we want this documentation to be so good that if you were to all leave the project tomorrow, someone else could read this and know exactly what to do to keep working on the project. That's quite a high bar so we won't always achieve that, but it's what we aim for. If there's something which for some reason can't be fully documented, e.g if some permissions or something needs to be handed over manually, then we should at least mention it here and make it clear who they can talk to about it.

Does that make sense? 🙂

Reply:
It would work most of the time, but also probably break mysteriously if people share sheets. We're trying to use Google Sheets as an ACID database, but Sheets doesn't really offer any way of controlling "transactions" in the "database". If multiple users were trying to edit/delete events at the same time then things may behave in ways that neither user was expecting. Instead I think they would have to coordinate amongst themselves to make sure only one person was running their code with that sheet at a time. Having a sheet per person would make things a lot more straightforward 🙂

There's two general steps when it comes to making a new sheet: setting the sheet up and setting a service account up. There could be one shared service account (set of credentials and an API key basically), and it could just be added to each of these new sheets. The downside to this is that the API quota will then be stretched across multiple people's worth of tests. You could also make a new service account and add it to the sheet, which means you wouldn't be sharing an API quota, but then you would also need access to the Google Cloud project where all of these credentials are stored. There isn't a seamless way of sharing these without GSuite accounts, though I'm not aware of a Softwire GSuite accounts but they might exist. Otherwise they would need to be manually added by a manager of the Google project (which I am the only one of at the moment).

Does that all make sense? Which approach do you think would work best? 🙂

Reply:
I think it's fine to keep the approach you have now, as long as it's well documented and easy to hand over. For example if you're the only admin at the moment then you need to make sure you hand that over. Ideally a shared account would be used as the admin. If it's free you can probably just set it up with the team DL. If it's not free you probably need a helpdesk ticket. Does that make sense? 🙂